.github/workflows/publish-docs.yml

```diff
@@ -49,6 +36,20 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install mkdocs
+          pip install git+https://${{ secrets.gh_token }}@github.com/lensapp/mkdocs-material-insiders.git
```

Comment:
Are you sure that `gh_token` can be lower case here?

Reply:
It has been like that since forever, but I'll change it anyway.

Reply:
Whoops, looks like I somehow managed to downcase it.